spec/models/case_contact_report_spec.rb

```diff
@@ -315,7 +315,8 @@
         it "returns all case contacts with the casa case ids" do
           report = described_class.new({casa_case_ids: [casa_case.id]})
           expect(report.case_contacts.length).to eq(case_contacts.length)
-          expect(report.case_contacts).to eq(case_contacts)
+          expect(report.case_contacts).to include(*case_contacts)
```

Comment:
@FireLemons here are the two adjustments that work, do you have a preference of what is more appropriate?

Reply:
Seems reasonable

Reply:
there is also `match_array` which should match the contents of an array without worrying about the order
As long as the code stays working and the test stops flaking I think we're good

Reply:
I vote for `match_array`!

Reply:
@compwron @littleforest done! :)